includes/wc-core-functions.php

```diff
@@ -470,9 +470,10 @@ function get_woocommerce_currency() {
  *
  * Currency symbols and names should follow the Unicode CLDR recommendation (http://cldr.unicode.org/translation/currency-names)
  *
+ * @param $sort Optionally allow sorting by currency title. Accepts asc, desc.
  * @return array
  */
-function get_woocommerce_currencies() {
+function get_woocommerce_currencies($sort = '') {
```

Comment:
It's already possible to filter and sort using `woocommerce_currencies`, so you can sort already for all your store instead of